eu.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "gresistor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST}",
+                "install -Dm644 gresistor.desktop  -t ${FLATPAK_DEST}/share/applications",
```

Comment:
you will need to add a `"rename-desktop-file": "gresistor.desktop",` above line 7.